doc/source/whatsnew/v0.19.0.txt

```diff
@@ -439,6 +439,17 @@ Deprecations
 - ``buffer_lines`` has been deprecated in ``pd.read_csv()`` and will be removed in a future version (:issue:`13360`)
 - ``as_recarray`` has been deprecated in ``pd.read_csv()`` and will be removed in a future version (:issue:`13373`)
 - top-level ``pd.ordered_merge()`` has been renamed to ``pd.merge_ordered()`` and the original name will be removed in a future version (:issue:`13358`)
+- The ``chunksize``, ``dayfirst``, ``doublequote``, ``escapechar``, ``lineterminator``, ``quotechar``, ``skipinitialspace``,
```

Comment:
make this a separate section. add to the highlites (single line with reference to this section).